File: C++/A/86.cpp
/*
シカのAtCoDeerくんは二つの正整数 a,b を見つけました。 a と b の積が偶数か奇数か判定してください。
*/
#include <iostream>
using namespace std;
int main()
{
    int a, b;
    cin >> a >> b;
    if ((a * b) % 2 != 0)
        cout << "Odd\n";
    else
        cout << "Even\n";
}

File: ruby/A/86.rb
# シカのAtCoDeerくんは二つの正整数 a,b を見つけました。 a と b の積が偶数か奇数か判定してください。
a, b = gets.split.map(&:to_i)
if (a * b) % 2 == 0
  puts "Even"
else
  puts "Odd"
end
# 01'00
